### PR TITLE
HD-108695 Freedom.tm > GA not tracking properly

### PR DESCRIPTION
**Pull request by wildtron**
## Ticket references
- [HD-108695 Freedom.tm > GA not tracking properly](https://freedom.myjetbrains.com/youtrack/issue/HD-108695)

## Summary of changes

* feat: add gtm support

## Checklist

- [ ] bugfix
- [x] new feature
- [ ] breaking change
- [ ] added unit tests
- [ ] changes introduced has been discussed and approved
- [ ] requires manual testing
- [ ] documentation has been updated

### CODE REVIEW

**Review by wilbertverayin — DISMISSED** [review]

**Review by pandafox — COMMENTED** [review]
I think the PR title/ticket is not related to GTM change?

File: src/router/index.js [hunk 1]
```diff
@@ -21,4 +22,15 @@ const Router = new VueRouter({
     routes
 });
 
+Router.afterEach(to => {
+    Vue.gtm.trackView(
+        to.path,
+        to.name,
+        {
+            cid: ga.getSessionId() || ga.createSessionId(),
```
Comment (pandafox) [inline src/router/index.js:30]: I don't think these files were checked in?

Comment (cedricabuso) [reply src/router/index.js:30]: @wildtron is `./analytics` an already existing file?

**Comment by jonelrempis** [comment]
@wildtron Ticket referenced was different from the changes?

File: src/router/index.js (hunk 1 above)
Comment (pandafox) [reply src/router/index.js:30]: With your latest code, we just need createSessionId()?

**Comment by wildtron** [comment]
> @wildtron Ticket referenced was different from the changes?

@jonelrempis Right. I was so sleepy when I was reading how to link GTM+GA properly then created this PR last night :sleepy: . I'll update.

File: src/router/index.js (hunk 1 above)
Comment (wildtron) [reply src/router/index.js:30]: Yeah. I'll have to resolve every comments first in FPD so we can finalize everything that's going to boilerplate.

Comment (wildtron) [reply src/router/index.js]: Resolved: https://github.com/anyTV/quasar-boilerplate/pull/32/commits/af5f421f6e2a99a589f082938cd1dacaf2d66365

File: quasar.conf.js [hunk 2]
```diff
@@ -14,8 +14,8 @@ module.exports = function (ctx) {
             // 'axios',
             // 'google-api',
             // 'jwt',
-            // 'vue-analytics',
             // 'vuelidate',
+            // 'vue-gtm'
```
Comment (pandafox) [inline quasar.conf.js:18]: Let's have this always enabled?

**Review by pandafox — APPROVED** [review]

File: quasar.conf.js (hunk 2 above)
Comment (wildtron) [reply quasar.conf.js]: https://github.com/anyTV/quasar-boilerplate/pull/32/commits/0cacf6a1b4aab80396a4b5b0cc4a3fde11929c70

**Review by pandafox — DISMISSED** [review]

**Review by wilbertverayin — DISMISSED** [review]

**Pull request merged**